cognition_objects/message.py

```diff
@@ -189,3 +198,147 @@ def delete(project_id: str, message_id: str, with_commit: bool = True) -> None:
         CognitionMessage.id == message_id,
     ).delete()
     general.flush_or_commit(with_commit)
+
+
+def get_messages_per_conversation_query(project_id: str) -> List[Dict[str, Any]]:
+    project_id = prevent_sql_injection(project_id, isinstance(project_id, str))
+    query = f"""
+    SELECT 
+        conversation_id,
+        c."header",
+        COUNT(*) messages
+    FROM cognition.message as m
+        INNER JOIN cognition.conversation c ON c.id = m.conversation_id AND c.project_id = m.project_id 
```

Comment:
doesnt match out sql style guide
```suggestion
    FROM cognition.message as m
    INNER JOIN cognition.conversation c 
        ON c.project_id = m.project_id AND c.id = m.conversation_id
```